types/appointment.d.ts

```diff
@@ -0,0 +1,62 @@
+declare module 'skedify-sdk' {
+  export interface Appointment extends Omit<BaseFields, 'enterprise_id'> {
+    state: string
+    contact_id: number
```

Comment:
Can you update the FKs to string? SDK Parses these as string

https://github.com/skedify/sdk-js/blob/6a10b3acf26dac6655f8c4a2d53488f7400ede63/src/decorators/withResources/normalizeResponse.js#L27